docs/src/modules/ROOT/pages/upgrade-and-migration/upgrade-to-latest-version.adoc

```diff
@@ -0,0 +1,213 @@
+[#upgradeToLatestVersion]
+= Upgrade to the latest version
+:doctype: book
+:sectnums:
+:icons: font
+
+Timefold Solver public APIs are backwards compatible,
+but users often also use impl classes,
+many of which are documented in the reference manual too.
```

Comment:
Do we have a link for that?

Reply:
There are many things like that. Move selectors, especially. Anyway, I'll rephrase it.